static/_redirects

```diff
@@ -1,7 +1,6 @@
 # see https://docs.netlify.com/routing/redirects/ for documentation
 # Redirects (temporary)
 /blog/:year/:month/:day/:slug                           /changelog/:slug                                    301
```

Comment:
We should be able to keep this, because new blog posts will never go that deep, I suppose. so, if a link to the blog is that deep, we can consider it's the old changelog